xds/src/main/java/io/grpc/xds/WeightedRoundRobinLoadBalancer.java

```diff
@@ -462,7 +459,10 @@ long getSequence() {
      * an offset that varies per backend index is also included to the calculation.
      */
     int pick() {
+      int i = 0;
```

Comment:
Clean up.

Reply:
Idea for checking this during tests: take the difference of AtomicInteger before/after calling this method. Probably best not to do that in this PR.